java/README.md

```diff
@@ -1,85 +1,65 @@
+It is sponsored and supported by AWS, and comes pre-configured with best practices learned from over a decade of operating Valkey-compatible services used by hundreds of thousands of customers.
+To help ensure consistency in development and operations, GLIDE for Valkey is implemented using a core driver framework, written in Rust, with extensions made available for each supported programming language. This design ensures that updates easily propagate to each language and reduces overall complexity.
+In this release, GLIDE for Valkey is available for Python, Javascript (Node.js), and Java.
+
+## Supported Valkey Versions
+
+GLIDE for Valkey is API-compatible with open source Valkey version 6 and 7.
+
+## Current Status
+
+We've made GLIDE for Valkey an open-source project, and are releasing it in Preview to the community to gather feedback, and actively collaborate on the project roadmap. We welcome questions and contributions from all Valkey stakeholders.
+This preview release is recommended for testing purposes only.
+
 # Getting Started - Java Wrapper
 
-## Notice: Java Wrapper - Work in Progress
+## System Requirements
+
+The beta release of GLIDE for Valkey was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7.
 
-We're excited to share that the Java client is currently in development! However, it's important to note that this client
-is a work in progress and is not yet complete or fully tested. Your contributions and feedback are highly encouraged as
-we work towards refining and improving this implementation. Thank you for your interest and understanding as we continue
-to develop this Java wrapper.
+## Supported Operating Systems
+
+GLIDE for Valkey is supported in Ubuntu, CentOS, and MacOS.
 
 The Java client contains the following parts:
 
-1. `client`: A Java-wrapper around the rust-core client.
-2. `examples`: An examples app to test the client against a Redis localhost
-3. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other Java clients.
-4. `integTest`: An integration test sub-project for API and E2E testing
+1. `src`: Rust dynamic library FFI to integrate with [GLIDE core library](https://github.com/aws/glide-for-redis/blob/main/glide-core/README.md).
+2. `client`: A Java-wrapper around the GLIDE core rust library and unit tests for it.
+3. `examples`: An examples app to test the client against a Valkey localhost.
+4. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Valkey and other Java clients.
+5. `integTest`: An integration test sub-project for API and E2E testing.
 
 ## Installation and Setup
 
 ### Install from Gradle
 
 At the moment, the Java client must be built from source.
 
-### Build from source
-
-Software Dependencies:
+#### Prerequisites
```

Comment:
We should mention that an existing Valkey-compatible server is required to run the integration tests. Probably shouldn't say "install redis".

Reply:
I believe IT expects redis-server to be installed.  
https://github.com/aws/glide-for-redis/blob/main/utils/cluster_manager.py#L287

Reply:
Wait, should we leave redis references in the code?

Reply:
until there's an equivalent setup for valkey-server, we don't have a choice.

Reply:
same change needed here: https://github.com/Bit-Quill/glide-for-redis/blob/main/python/DEVELOPER.md?plain=1#L113